interface/src/app/analyze/analysis/analysis.less

```diff
@@ -1,3 +1,37 @@
+/*
+ * styles to make sampleBin editing controls more user-friendly
+ */
+.row-control {
+  float: none;
+  .opacity(.2);
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+    cursor: pointer;
+    .opacity(.5);
+  }
+
+  button& {
+    padding: 0;
+    cursor: pointer;
+    background: transparent;
+    border: 0;
+    -webkit-appearance: none;
+  }
+}
+// allow us to use controls to help align headers, but don't show them
+th .row-control, thead .row-control {
+  visibility: hidden;
+}
+
+// sample group highlighting
```

Comment:
I like the highlight colors you set on the web interface. Do you think it will be helpful to describe the color roughly using comments?